gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```

Comment:
this project use android-gradle-plugin-3.1.2, it not support gradle 4.6 or above version yet, if we use gradle 4.6 or above version, it report a error as follow:

Configuration on demand is not supported by the current version of the Android Gradle plugin since you are using Gradle version 4.6 or abov e. Suggestion: disable configuration on demand by setting org.gradle.configureondemand=false in your gradle.properties file or use a Gradle version less than 4.6.

resolve this problem have at least two methods:
1、use a Gradle version less than 4.6.
2、setting org.gradle.configureondemand=false in gradle.properties file

configure on demand may speed up builds, and Gradle4.4 is enough, so use a Gradle4.4 is best choice.

Reply:
It is best to stay on the latest version of all tools. It looks like it was deprecated because it isn't actually necessary with newer versions. Please disable it and upgrade gradle instead.
https://github.com/gradle/gradle/issues/2868

Reply:
I will revert the version of gradle to 4.7